translations/pl.json

```diff
@@ -947,20 +947,20 @@
 	"{{.driver_name}} couldn't proceed because {{.driver_name}} service is not healthy.": "",
 	"{{.driver_name}} has less than 2 CPUs available, but Kubernetes requires at least 2 to be available": "",
 	"{{.driver_name}} has only {{.container_limit}}MB memory but you specified {{.specified_memory}}MB": "",
-	"{{.driver}} only has {{.size}}MiB available, less than the required {{.req}}MiB for Kubernetes": "",
-	"{{.extra_option_component_name}}.{{.key}}={{.value}}": "",
+	"{{.driver}} only has {{.size}}MiB available, less than the required {{.req}}MiB for Kubernetes": "sterownik {{.driver}} ma tylko {{.size}}MiB dostępnej przestrzeni dyskowej, to mniej niż wymagane {{.req}}MiB dla Kubernetesa",
+	"{{.extra_option_component_name}}.{{.key}}={{.value}}": "{{.extra_option_component_name}}.{{.key}}={{.value}}",
```

Comment:
If there's no difference between the translated version can be left empty, but doesn't matter either way